tests/phpunit/integration/Core/Util/SortTest.php

```diff
@@ -0,0 +1,90 @@
+<?php
+/**
+ * SortTest
+ *
+ * @package   Google\Site_Kit\Tests\Core\Util
+ * @copyright 2022 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Tests\Core\Util;
+
+use Google\Site_Kit\Core\Util\Sort;
+use Google\Site_Kit\Tests\TestCase;
+
+/**
+ * @group Util
+ */
+class SortTest extends TestCase {
+	/**
+	 * Sample unsorted array.
+	 * @var array
+	 */
+	private static $unsorted_array = array(
```

Comment:
I would not use static class variables for these test inputs and outputs, for a couple of reasons.

For one thing, if this file grows, these variables might end up only being used in the one test case, and so the organisation value of extracting to class variables diminishes.

Secondly, as they are only instantiated once, these are stateful and liable to change over the lifetime of the test suite execution. For example, `$unsorted_array` is being provided as input into two tests. Say there's an error in the first test which results in `$unsorted_array` being modified. This would have a knock-on effect to the second test as the input would now be different to what the test is designed for.

The potential for accidental mutation of test data is something that is best avoided, either by creating the data freshly for each test, or by using immutable data, where possible (e.g. `Object.freeze` in JS-land).

In this specific case, I think we can simply move these arrays within the test case itself, and avoid passing `$unsorted_array` into two tests by virtue of removing one of them as per the comment above.

Reply:
Updated, thank you for the clarification!